hrui/tests/phpunit/CRM/HRUI/HelperTest.php

```diff
@@ -14,21 +14,13 @@ class CRM_HRUI_HelperTest extends \PHPUnit_Framework_TestCase implements Headles
 
   use HRUITrait;
 
-  protected $requiredExtensions = [
-    'uk.co.compucorp.civicrm.hrcore', // required for fabricator
-    'org.civicrm.hrident', // creates the Identity custom group required by hrcase
-    'uk.co.compucorp.civicrm.tasksassignments', // if not enabled will try to redirect
-    'org.civicrm.hrcase' // creates the line manager relationship type
-  ];
-
   /**
    * @return CiviEnvBuilder
    */
   public function setUpHeadless() {
-    return \Civi\Test::headless()
-      ->install($this->requiredExtensions)
-      ->installMe(__DIR__)
-      ->apply();
+    CRM_HRCase_Upgrader::createRelationshipTypes();
```

Comment:
That makes sense, I just didn't want to duplicate the code, but it's a good point that calling specific upgrader functions is a bit messy. I'll copy the code to the test case.